doc/source/guide/acquiring_data/fido.rst

```diff
@@ -0,0 +1,68 @@
+-----------------------------
+Downloading Data using Fido
+-----------------------------
+
+This guide outlines how to search for and download data using SunPy's
+Federated Internet Data Obtainer...or more usually (and
+sanely) referred to as Fido.  Fido is a unified API for seeking
+and downloading solar physics data irrespective of the underlining
+client or webservice through which the data is obtained, e.g. VSO,
+JSOC etc.  It therefore supplies a single easy and consistent way to
+obtain most forms of solar physics.
+
+Setup
+-----
+
+SunPy's Fido module is in ``sunpy.net``.  It can be imported as follows:
+
+    >>> from sunpy.net import Fido
+
+Searching for Data Using Fido
+-----------------------
+
+To search for data with Fido, your query needs at minimum a start time,
+an end time, and an instrument.  Enter these properties using SunPy's
+attrs module.
+
+    >>> from sunpy.net import attrs as a
+    >>> unifresp = Fido.search(a.Time('2012/3/4', '2012/3/6'), a.Instrument('lyra'))
+
+This returns an UnifiedResponse object containing information on the
+available online files which fit the criteria specified by the attrs
+objects in the above call.  It does not download the files.  For
+downloading, see the Downloading Data with Fido section below.
```

Comment:
yes you have to define a id http://www.sphinx-doc.org/en/stable/markup/inline.html#ref-role